src/HTCondor/condor-9-to-10-upgrade-check.py

```diff
@@ -327,40 +365,33 @@ def check_pcre2():
     if platform.system() == "Linux" and "el7" in platform.release():
         is_el7 = True
     format_print("Checking for incompatibilities with PCRE2.", newline=True)
-    format_print("Known PCRE2 incompatibilities:", 8)
-    format_print("1. Hyphens in a regex character sequence must occur at the end.", 16)
+    format_print("Known PCRE2 incompatibilities:", offset=8)
+    format_print("1. Hyphens in a regex character sequence must occur at the end.", offset=16)
     if is_el7:
         format_print(
-            "2. On EL7 posix name sets ([:space:]) followed by hyphen are invalid.", 16
+            "2. On EL7 posix name sets ([:space:]) followed by hyphen are invalid.", offset=16
         )
     # Check for 'pcre2grep' cmd
     try:
         p = subprocess.run(["pcre2grep"], stderr=subprocess.PIPE)
         has_pcre2_cmd = True
     except FileNotFoundError:
-        if is_el7:
-            format_print(
-                "Failed to find 'pcre2grep' command. Only checking for known issue #2.",
-                8,
-                err=True,
-            )
-        else:
-            format_print(
-                "Failed to find 'pcre2grep' command. Unable to check Map files for incompatibilties.",
-                8,
-                err=True,
-            )
-            return
+        format_print(
+            """Failed to find 'pcre2grep' command. Will do a simple check for incompatibilties.
+        Recommended to install 'pcre2grep' command for better check of map files.""",
+            offset=8,
+            err=True,
+        )
```

Comment:
I didn't even think about what `offset` would do with multiline messages, but testing shows this works right. I might suggest just omitting it for multiline messages, though, and just put whatever spaces are needed in the string itself:

```suggestion
        format_print(
            """        Failed to find 'pcre2grep' command. Will do a simple check for incompatibilties.
        Recommended to install 'pcre2grep' command for better check of map files.""",
            err=True,
        )
```